### PR TITLE
Adding encontrado column and missing days

### PR DESCRIPTION
**Pull request by brianstanley**
Also removing social icons and ultimo avistamiento
issue #44

### CODE REVIEW

File: src/components/main/person/PersonFoundList.vue [hunk 1]
```diff
@@ -75,7 +76,7 @@
           this.selectedItem    = {};
           this.showDeleteModal = false;
         });
-      }
+      },
```
Comment (ndelvalle) [inline src/components/main/person/PersonFoundList.vue]: Extra semicolon

Comment (ndelvalle) [inline src/components/main/person/PersonFoundList.vue]: Extra semicolon?

Comment (brianstanley) [inline src/components/main/person/PersonFoundList.vue]: oh yes. Removed

File: src/components/main/person/PersonFoundList.vue [hunk 2]
```diff
@@ -116,6 +120,8 @@
             td {{ person.geo.city }}
             td {{ person.createdAt | moment 'from' }}
             td {{ person.createdAt | moment shortDate }}
+            td {{ person.foundAt   | moment shortDate }}
+            td {{ person.foundAt   | moment "from" person.lastSeen true }}
```
Comment (ianaya89) [inline src/components/main/person/PersonFoundList.vue]: Use always single quotes `''`

**Pull request merged**